src/index.js

```diff
@@ -51,16 +51,37 @@ class App {
     this.container.appendChild(addTask);
     this.container.appendChild(this.taskListContainer);
     this.container.appendChild(buttonDeleteCompletedTasks);
+
+    if (localStorage.toDoTaskList.length) {
+      this.taskList = this.getTaskListFromLocalStorage();
```

Comment:
line 56 can be moved out of if statement to line 54 
checking if task list in localStorage should be done in getTaskListFromLocalStorage 
if ls empty, [] can be returned

Reply:
done